<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTask</class>
 <widget class="QWidget" name="NewTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>280</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>495</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>New Task</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxTags">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Tags</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>76</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtDescription">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCreate">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>94</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>94</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>109</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Due date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dteDate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>351</width>
     <height>26</height>
    </rect>
   </property>
   <property name="date">
    <date>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>MM/dd/yy h:mm AP</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
